client/src/components/layouts/global.css

```diff
@@ -1,10 +1,16 @@
 .btn-cta-big {
   max-height: 100%;
-  height: 70px;
+  height: auto;
   font-size: 40px;
   white-space: normal;
 }
 
+@media (min-width: 768px) {
```

Comment:
I have been thinking that starting from `769px` would make more sense, but I noticed that in the codebase we already have 768: https://github.com/freeCodeCamp/freeCodeCamp/blob/1f923db64b08d55deab4cca4be105efbe5b23163/client/src/templates/Challenges/components/video-modal.css#L5

Reply:
Is line 8 to 13 needed? If yes, why? because height: auto will do the trick in all device size

Reply:
@rupali317 
I've checked on production website, and indeed height is missing there for those buttons.
But, the `height: 70px` was added for some reason. I would like to hear from dev team whether or not they want to keep 70px height limit or we should be okay with `height: auto`.

Reply:
It was probably me that added the height rule, and css isn't my strong point.

If what you propose works, then I am fine about changing it. My code is not precious to me.

Reply:
`70px` makes sense to me for desktop devices.

Reply:
Even in desktop devices i doubt 70px will be correct as users can always resize the browser and we may see this issue again

Perhaps you can test only with height: auto and it should be fine. Let me know if that works

Reply:
@rupali317 
Let me check

Reply:
height: auto fixes it on my side without the media queries.

Reply:
Updated the PR